parcels/codegenerator.py

```diff
@@ -38,10 +38,11 @@ def __getattr__(self, attr):
 class RandomNode(IntrinsicNode):
     symbol_map = {'random': 'parcels_random',
                   'uniform': 'parcels_uniform',
-                  'randint': 'parcels_randint'}
+                  'randint': 'parcels_randint',
+                  'normal': 'parcels_normal'}
 
     def __getattr__(self, attr):
-        if hasattr(random, attr):
+        if hasattr(random, attr) or attr is 'normal':
```

Comment:
Without this statement `or attr is 'normal'`, JIT mode throws an error
`AttributeError: Unknown random function encountered: normal`

Not sure how to fix this...